data/json/items/tools.json

```diff
@@ -2940,38 +2940,141 @@
     "use_action": "MOLOTOV_LIT"
   },
   {
-    "id": "acidbomb",
```

Comment:
You can't just delete an item like that or saves will break. You can move it into obsolete.json or change one of the bombs' id to just "acidbomb".

An item with id "acidbomb" has to exist, even if just in obsolete.json.